README.md

```diff
@@ -1 +1,92 @@
-.
+<h1 align='center'>
+    PubSubs
+</h1>
+
+<h4 align='center'>
+    Uniform interfaces for Publishing and Subscribing
+</h4>
+
+---
+
+This library allows you to decouple the publish/subscribe implementation
+from your service.
+
+## Installing
+
+```
+pip install pubsubs[<implementation-name>]
+```
+
+## Usage
+
+### Publisher
+
+New Kafka publisher instance
+
+```python
+from pubsubs.registry import Registry
+
+config = {
+    "listeners": ['localhost:9092'],
+    "publisher": {
+        "poll": 1.0,
+        "message.timeout.ms": 1500,
+    }
+}
+
+registry = Registry()
+
+kafka = registry.new("kafka1", backend="kafka", **config)
+
+kafka.publish(topic="MyTopic", message="hey!")
+```
+
+### Subscriber
+
+Configuring a Kafka subscriber and using a yaml.
+
+```yaml
+pubsubs:
+    myKafka:
+        backend: kafka
+        listeners: ['localhost:9092']
```

Comment:
Do publishers and subscribers always use the same `listeners`?

Reply:
I'd say yes. If you wanted to configure a publisher separately you could do this:

```yaml
pubsubs:
    kafkaPub:
        backend: kafka
        listeners: ['localhost:9092']
        publisher:
            poll: 1.0
            message.timeout.ms: 1500
    kafkaSub:
        backend: kafka
        listeners: ['localhost:9093']
        subscriber:
            poll: 0.1
            group.id: 'myGroup'
            auto.offset.reset: 'earliest'
```